app/utils/googleDocs.scala

```diff
@@ -31,33 +34,31 @@ object CSV extends RegexParsers {
   def field: Parser[String] = escaped|nonescaped
 
   def escaped: Parser[String] = {
-    ((SPACES?)~>DQUOTE~>((TXT|COMMA|CRLF|DQUOTE2)*)<~DQUOTE<~(SPACES?)) ^^ {
-      case ls => ls.mkString("")
-    }
+    ((SPACES?)~>DQUOTE~>((TXT|COMMA|CRLF|DQUOTE2)*)<~DQUOTE<~(SPACES?)) ^^ (ls => ls.mkString(""))
   }
 
-  def nonescaped: Parser[String] = (TXT*) ^^ { case ls => ls.mkString("") }
+  def nonescaped: Parser[String] = (TXT*) ^^ (ls => ls.mkString(""))
 
-  def parse(s: String) = parseAll(file, s) match {
+  def parse(s: String): List[List[String]] = parseAll(file, s) match {
     case Success(res, _) => res
     case e => throw new Exception(e.toString)
   }
 }
 
 object GoogleDoc extends Logging {
```

Comment:
As mentioned, pretty sure this is no longer used so let's remove it.

Reply:
✅  in [this commit](https://github.com/guardian/prism/pull/84/commits/f133b8649cfd6e8e90f3072c24386fdcb56347e3)